app/views/batch_contexts/_new_bc_form.erb

```diff
@@ -20,6 +20,8 @@
         ['File', 'file'],
         ['Media', 'media'],
         ['3D', '3d'],
+        ['Map', 'maps'],
```

Comment:
Same comment as above, re: maps vs map

Reply:
Renamed 'simple image' and 'simple book' to 'image' and 'book' in the UI to match what is in Argo.